### PR TITLE
[Portuguese] Translate guide/getting-started.md

### PR DESCRIPTION
**Pull request by roziscoding**

### CODE REVIEW

File: site/docs/pt/guide/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+```asciiart:no-line-numbers
+.
+├── bot.ts
+├── node_modules/
+├── package.json
+├── package-lock.json
+└── tsconfig.json
+```
+
+Agora, é hora de abrir o Telegram para criar uma conta de bot e obter um token de bot para ele.
```
Comment (kr4t0z) [inline site/docs/pt/guide/getting-started.md:68]:
```suggestion
Agora, é hora de abrir o Telegram para criar uma conta de bot e obter um token de bot para ela.
```
I think the personal pronoun here is referring to "conta de bot", so it should be in the femine form.

File: site/docs/pt/guide/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+import { Bot } from "grammy";
+
+// Crie uma instância da classe `Bot` e passe seu token de bot para ela.
+const bot = new Bot(""); // <-- coloque o token do seu bot entre ""
+
+// Agora você pode registrar listeners no objeto `bot`.
+// O grammY irá chamar os listeners quando usuários enviarem mensagens para seu bot.
+
+// Trata o comando /start.
+bot.command("start", (ctx) => ctx.reply("Boas vindas! Online e roteando 😎"));
```
Comment (kr4t0z) [inline site/docs/pt/guide/getting-started.md:88]:
```suggestion
bot.command("start", (ctx) => ctx.reply("Boas vindas! Online e roteando :sunglasses:"));
```

File: site/docs/pt/guide/getting-started.md (hunk 1 above)
Comment (roziscoding) [reply site/docs/pt/guide/getting-started.md:68]: I agree! I didn't like how this sounded initially too, but didn't really know what to do with it. Thanks!

File: site/docs/pt/guide/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+└── tsconfig.json
+```
+
+Agora, é hora de abrir o Telegram para criar uma conta de bot e obter um token de bot para ele.
+Fale com o [@BotFather](https://t.me/BotFather) para fazer isso.
+O token do bot é algo como `123456:aBcDeF_gHiJkLmNoP-q`.
+Ele é usado para autenticar seu bot.
+
+Pegou o token? Agora você pode escrever o código do seu bot no arquivo `bot.ts`.
+Você pode copiar o seguinte bot de exemplo para esse arquivo, e passar seu token para o construtor `Bot`:
```
Comment (roziscoding) [inline site/docs/pt/guide/getting-started.md:74]:
```suggestion
Copie o seguinte bot de exemplo para esse arquivo, e passar seu token para o construtor `Bot`:
```

File: site/docs/pt/guide/getting-started.md [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+touch bot.ts
+```
+
+Agora, é hora de abrir o Telegram para criar uma conta de bot e obter um token de bot para ele.
+Fale com o [@BotFather](https://t.me/BotFather) para fazer isso.
+O token do bot é algo como `123456:aBcDeF_gHiJkLmNoP-q`.
+Ele é usado para autenticar seu bot.
+
+Pegou o token? Agora você pode escrever o código do seu bot no arquivo `bot.ts`.
+Você pode copiar o seguinte bot de exemplo para esse arquivo, e passar seu token para o construtor `Bot`:
```
Comment (roziscoding) [inline site/docs/pt/guide/getting-started.md:171]:
```suggestion
Copie o seguinte bot de exemplo para esse arquivo, e passar seu token para o construtor `Bot`:
```

File: site/docs/pt/guide/getting-started.md [hunk 5]
```diff
@@ -0,0 +1,222 @@
...
+import { Bot } from "https://deno.land/x/grammy/mod.ts";
+
+// Crie uma instância da classe `Bot` e passe seu token de bot para ela.
+const bot = new Bot(""); // <-- coloque o token do seu bot entre ""
+
+// Agora você pode registrar listeners no objeto `bot`.
+// O grammY irá chamar os listeners quando usuários enviarem mensagens para seu bot.
+
+// Trata o comando /start.
+bot.command("start", (ctx) => ctx.reply("Boas vindas! Online e roteando 😎"));
```
Comment (kr4t0z) [inline site/docs/pt/guide/getting-started.md:183]:
```suggestion
bot.command("start", (ctx) => ctx.reply("Boas vindas! Online e roteando :sunglasses"));
```

File: site/docs/pt/guide/getting-started.md [hunk 6]
```diff
@@ -0,0 +1,222 @@
...
```
Comment (roziscoding) [inline site/docs/pt/guide/getting-started.md:183]:
```suggestion
bot.command("start", (ctx) => ctx.reply("Boas vindas! Online e roteando :sunglasses:"));
```

**Review by kr4t0z — APPROVED** [review]
awesome work 💪

File: site/docs/pt/guide/getting-started.md [hunk 7]
```diff
@@ -0,0 +1,222 @@
...
+import { Bot } from "grammy";
+
+// Crie uma instância da classe `Bot` e passe seu token de bot para ela.
+const bot = new Bot(""); // <-- coloque o token do seu bot entre ""
+
+// Agora você pode registrar listeners no objeto `bot`.
+// O grammY irá chamar os listeners quando usuários enviarem mensagens para seu bot.
+
+// Trata o comando /start.
+bot.command("start", (ctx) => ctx.reply("Boas vindas! Online e roteando :sunglasses:"));
```
Comment (roziscoding) [inline site/docs/pt/guide/getting-started.md:88]:
```suggestion
bot.command("start", (ctx) => ctx.reply("Online e roteando :sunglasses:"));
```

File: site/docs/pt/guide/getting-started.md [hunk 8]
```diff
@@ -0,0 +1,222 @@
...
+import { Bot } from "https://deno.land/x/grammy/mod.ts";
+
+// Crie uma instância da classe `Bot` e passe seu token de bot para ela.
+const bot = new Bot(""); // <-- coloque o token do seu bot entre ""
+
+// Agora você pode registrar listeners no objeto `bot`.
+// O grammY irá chamar os listeners quando usuários enviarem mensagens para seu bot.
+
+// Trata o comando /start.
+bot.command("start", (ctx) => ctx.reply("Boas vindas! Online e roteando :sunglasses:"));
```
Comment (roziscoding) [inline site/docs/pt/guide/getting-started.md:183]:
```suggestion
bot.command("start", (ctx) => ctx.reply("Online e roteando! :sunglasses:"));
```

**Review by kr4t0z — APPROVED** [review]
I'll miss the "Boas vindas" 🥲
But it's better. Let's merge

**Pull request merged**